src/main/resources/application.conf

```diff
@@ -257,7 +257,7 @@ grothendieck {
     sync-retry-interval = 5.seconds
 
     # Response time-out from peer during sync. If a peer fails to respond within this limit, it will be blacklisted
-    peer-response-timeout = 10.seconds
+    peer-response-timeout = 3.minutes
```

Comment:
I needed to set it to 2 and it worked all the time, but cannot remember if tried with 1 first :thinking: